src/data/markdown/docs/03 cloud/05 Project and Team Management/03 Members.md

```diff
@@ -32,13 +32,13 @@ Gives a project member the ability to create, edit, run and schedule tests in a
 
 #### Read-Only
 
-Gives a project member the ability only to view tests and test runs in a particular project, not being able to modify them in any way.
+Gives a project member the ability only to view tests and test runs in a particular project. They are not able to modify them in any way.
 
-> ⚠️ &nbsp; Set the project ID when running a cloud test from the k6 CLI. Read more [here](/cloud/project-and-team-management/projects/#running-cli-tests-in-a-specific-project).  
+> ⚠️ &nbsp; If you only have access to run tests in a specific project, don't forget to specify its projectID when running a cloud test from the k6 CLI. Read more [here](/cloud/project-and-team-management/projects/#running-cli-tests-in-a-specific-project).
 
 ## Managing members
 
-There's two places in the k6 Cloud app where you can manage members - [Members settings](/cloud/project-and-team-management/members/#members-settings) under the **Organization settings** or through [Project settings](/cloud/project-and-team-management/members/#project-settings). 
+There's two places in the k6 Cloud app where you can manage members - [Members settings](/cloud/project-and-team-management/members/#members-settings) under the **Organization settings** or through [Project settings](/cloud/project-and-team-management/members/#project-settings).
```

Comment:
(This and later diffs are just the editor killing the whitespace at the end of the line, no other changes)